themes/pcw-hugo-theme/layouts/partials/i18nlist.html

```diff
@@ -1,8 +1,8 @@
 {{ if .IsTranslated }}
 <ul id="translations" class="pl0 mr3-l db-l dn">
     {{ range .Translations }}
-    <li class="list f4 fw4 dib pr3-l pt2 pb1">
```

Comment:
This caused padding-right to be applied to the "español" link, which made the background hover effect look weird.

I couldn't think of any reason why the padding would be necessary -- I tried it at a few different screen sizes and it looked fine, but if anyone knows better we can undo it.